base/show.jl

```diff
@@ -1128,6 +1128,8 @@ function isidentifier(s::AbstractString)
 end
 isidentifier(s::Symbol) = isidentifier(string(s))
 
+is_op_suffix_char(c::AbstractChar) = ccall(:jl_op_suffix_char, Cint, (UInt32,), c) != 0
```

Comment:
This function isn't annotated with `JL_DLLEXPORT`: https://github.com/JuliaLang/julia/blob/7ecd1f9e0da5d07e7dd1733093b246a37bc118ff/src/flisp/julia_extensions.c#L152. Do I need to add that or is this ok?

Reply:
As @giordano pointed out on Slack, this is not strictly needed, because `jl_op_suffix_char` is exported in `flisp.h` already, but for consistency with `jl_id_char` and `jl_id_start_char`, I think it makes sense to add this here as well.